schemas/HPXMLBaseElements.xsd

```diff
@@ -1068,6 +1068,9 @@
 									<xs:element minOccurs="0" name="Emittance" type="Emittance"/>
 									<xs:element minOccurs="0" name="InteriorFinish" type="InteriorFinishInfo"/>
 									<xs:element minOccurs="0" maxOccurs="1" name="Insulation" type="InsulationInfo"/>
+									<xs:element name="RadiantBarrier" type="HPXMLBoolean" minOccurs="0"/>
```

Comment:
I suggest moving these elements above Insulation, for consistency with the Roof element.